examples/notebooks/data_store_sqlite.ipynb

```diff
@@ -20,7 +22,7 @@
     "import os\n",
     "import sys\n",
     "\n",
-    "module_path = os.path.abspath(os.path.join('..'))\n",
+    "module_path = os.path.abspath(os.path.join('../../'))\n",
```

Comment:
Actually, I haven't checked this cell after migrating to the new repository. Have you tried to run the notebook without it? Was it necessary to add module paths in `../..` ?

Reply:
I needed to do this to make it work.  The current file is an extra level down.

Reply:
Okay, PyCharm worked without it and therefore I wanted to ask. It is probably setting up the project environment by default.